instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+# Trainer Training Process
+
+This document describes the workflow for training new instructors,
+and is meant as both a guide to trainers
+and as a checklist for adding support for this workflow to [AMY][amy].
+
+## Workflow
+
+1.  **Recruiting candidates.**
+    1.  People may apply directly as individuals to become instructors
+        by filling in a form on our website.
+    1.  People may apply in physically co-located groups.
+        We have only done this one (for training in Dec 2015),
+	but it worked so well that we would like it to become routine.
+	In this case,
+	groups need to fit between a minimum and maximum size (e.g., 3 to 12)
+	and the application must includes names and contact information for everyone who intends to take part.
+	Groups must also commit to organizing and running at least one workshop within a few months of training.
+    1.  We may commit to training people for partner organizations as part of their partnership agreements.
+        Here,
+	we negotiate dates with the partner site,
+	which selects the participants subject to the same per-site size restrictions above (3-12 people per site).
+1.  **Matching candidates to courses.**
+    This has been done manually to date:
+    the lead trainer, members of the Steering Committee, and/or the Executive Directors
+    decide on dates and then invite individuals and/or groups to take part.
+1.  **Running courses.**
+    1.  In-person courses are run over two days,
+        with all participants at one site.
+    1.  Live online courses are also run over two days,
+        but participants take part in small groups at a small number of sites
+	(typically 3-12 per site and 2-4 sites).
+    1.  Asynchronous online courses are run over several weeks.
+        Participants are typically *not* co-located,
+	but get together for an hour of discussion online each week.
+	This format is being phased out,
+	as it took more effort from trainers
+	and had poorer follow-through rates than the intensive two-day versions.
```

Comment:
Is there anything (about *Matching candidates to courses* and *running courses*) that could be automated or made easier except for moving the list of partners from Google Spreadsheet to Amy?